src/Http/Headers/src/Properties/AssemblyInfo.cs

```diff
@@ -4,3 +4,9 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("Microsoft.Net.Http.Headers.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100f33a29044fa9d740c9b3213a93e57c84b472c84e0b8a0e1ae48e67a9f8f6de9d5f7f3d52ac23e48ac51801f1dc950abe901da34d2a9e3baadb141a17c77ef3c565dd5ee5054b91cf63bb3c6ab83f72ab3aafe93d0fc3c2348b764fafb0b1c0733de51459aeab46580384bf9d74c4e28164b7cde247f891ba07891c9d872ad2bb")]
+[assembly: InternalsVisibleTo("Microsoft.AspNetCore.Server.Kestrel.Core, PublicKey=0024000004800000940000000602000000240000525341310004000001000100f33a29044fa9d740c9b3213a93e57c84b472c84e0b8a0e1ae48e67a9f8f6de9d5f7f3d52ac23e48ac51801f1dc950abe901da34d2a9e3baadb141a17c77ef3c565dd5ee5054b91cf63bb3c6ab83f72ab3aafe93d0fc3c2348b764fafb0b1c0733de51459aeab46580384bf9d74c4e28164b7cde247f891ba07891c9d872ad2bb")]
```

Comment:
While we do use InternalsVisibleTo for tests, we almost never use it between production assemblies.

It looks like the only component outside of Kestrel.Core that needs these pseudo headers is http2cat? If that's all, http2cat can have its own copy and the main version can move into Kestrel.Core.

Reply:
Apparently, `Microsoft.Net.Http.Headers.Tests` also needs it, as CI just pointed out. Should I make a copy there as well? I guess we could update the CodeGenerator to make multiple copies and just place each one is the correct directory. Would this be a good solution?

Reply:
Where do those tests need them?

Reply:
Oh, I misunderstood, `Microsoft.Net.Http.Headers.Tests` doesn't need the pseudo-headers, but the dependency existed prior to this PR.